packages/graphql/src/services/stacks.ts

```
@@ -2178,9 +2179,9 @@ function computeAggregatedResponsesChoices({
   newAggResponses.choices = (
     response as SingleQuestionResponseChoices
   ).choices.reduce(
-    (acc, ix) => ({
+    (acc, choiceResponse) => ({
       ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)